index.html

```diff
@@ -64,5 +64,196 @@ <h2 class="intro connect">LET'S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <!-- Work Section -->
+    <section class="wrapper">
+      <div class="shot">
+        <div class="snapshot"><img src="images/Shot1.svg" alt="tonic" /></div>
+        <div class="block">
+          <h1 class="heading1">Tonic</h1>
+          <div class="subframe">
+            <p class="sub1">CANOPY</p>
+            <div class="circle"></div>
+            <p class="sub2">Back End Dev</p>
+            <div class="circle"></div>
+            <p class="sub2">2015</p>
+          </div>
+          <div class="normal">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </div>
+          <ul class="icons">
+            <li class="iconsub">html</li>
+            <li class="iconsub">css</li>
+            <li class="iconsub">javaScript</li>
+          </ul>
+          <div class="action">
+            <button class="button" type="button">See Project</button>
```

Comment:
- [ ] The padding of the `See Project` button appears to be wider than the Figma design. Kindly check.